### ISSUE TITLE
HiFi-GAN parameters

### ISSUE THREAD

**Issue by majidAdibian77** [kan-bayashi/ParallelWaveGAN#343]
Hi
Thanks for this great project.
I trained a multi speaker Tacotron using [this repo](https://github.com/CorentinJ/Real-Time-Voice-Cloning) in new dataset and results was good. Then I test hifigan as vocoder from [this repo](https://github.com/jik876/hifi-gan) (that is a official implementation) but result was very noisy so I think some parameters are different in hifigan and my synthesizer. 
Then I used hifigan from your repo and result is very good!
What is the difference between these two hfigan implementations?

**Comment by kan-bayashi** [comment]
Not sure but the following points might be different:
- log basis (I use log10 as a default)
- fmin and fmax for mel basis (I use 80-7600 as a default)
- normalization (I use mean var normalization using stats of training data).

**Comment by kan-bayashi** [comment]
> Another question that there are 3 versions of hifigan with different speed and number of parameters (v1, v2 and v3). The implemented hifigan in this repo is which one?

Existing config and pretrained models are based on v1. But v2 and v3 can be reproduced by changing hyperparameters.
